.github/labeler.yml

```diff
@@ -3,7 +3,7 @@ v7:
 
 documentation:
   - head-branch: ['^doc']
-  - changed-files: '**/*.md'
+  - changed-files: ['**/*.md']
```

Comment:
Is it can ignore test files like v6?

Reply:
snapshots in `v7` is under `__test__` and not a markdown files. So I think this condition should be enough.

Reply:
I see

Reply:
Snapshots on v7 are using `.snap.js` as well, it won't confusing for sure.